pkgs/development/python-modules/pytest/default.nix

```diff
@@ -23,7 +23,10 @@ let generic = { version, sha256 }:
       ++ stdenv.lib.optionals (!isPy3k) [ funcsigs ]
       ++ stdenv.lib.optionals (pythonOlder "3.6") [ pathlib2 ];
 
-    checkPhase = ''
+    checkPhase = if isPyPy then ''
```

Comment:
`doCheck = !isPyPy`?

Reply:
I wasn't sure if the hooks were necessary.